lte/gateway/release/build-magma.sh

```diff
@@ -130,6 +130,7 @@ OAI_DEPS=(
     "liblfds710"
     "magma-sctpd >= ${SCTPD_MIN_VERSION}"
     "libczmq-dev >= 4.0.2-7"
+    "libczmq-dbg >= 4.0.2-7"
```

Comment:
Will this add it to all Magma images? If so do we want to just do this on mme_testing?